var/spack/repos/builtin/packages/tiramisu/package.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import llnl.util.filesystem as fs
+from spack.package import *
+
+
+class Tiramisu(CMakePackage, PythonExtension):
+    """Tiramisu is a polyhedral compiler for dense and sparse deep learning
+    and data parallel algorithms.It provides a simple C++ API for expressing
+    algorithms and how these algorithms should be optimized by the compiler."""
+
+    homepage = "http://tiramisu-compiler.org"
+    url = "https://github.com/Tiramisu-Compiler/tiramisu"
+    git = "https://github.com/Tiramisu-Compiler/tiramisu.git"
+
+    maintainers = ["wraith1995"]
+
+    generator = "Ninja"
+
+    version("master", branch="master")
+    version("2023-2-3", commit="73a9cec72e08d4dfe5e8c66da33139008124a4fa")
+
+    variant("python", default=True, description="Install python bindings.")
+    variant("cuda", default=False, description="Enable Cuda Code Generation.")
```

Comment:
Might be good to subclass CudaPackage so you can get all of the other nice features that base class adds. Just add CudaPackage to the list of superclasses, and remove the variant and dependency since those are already added by the base class.

Reply:
Done